### ISSUE TITLE
Investigate failing Pallet UI tests

### ISSUE THREAD

**Issue by ggwpez** [paritytech/substrate#12955]
The pallet UI tests sometimes fail with just one line of diff. It often looks like this:  

```diff
@@ -28,7 +28,7 @@ error[E0277]: the trait bound `Bar: EncodeLike` is not satisfied
              <&[(T,)] as EncodeLike<BinaryHeap<LikeT>>>
              <&[(T,)] as EncodeLike<LinkedList<LikeT>>>
              <&[T] as EncodeLike<Vec<U>>>
-           and 278 others
+           and 279 others
    = note: required for `Bar` to implement `FullEncode`
    = note: required for `Bar` to implement `FullCodec`
    = note: required for `frame_support::pallet_prelude::StorageValue<_GeneratedPrefixForStorageFoo<T>, Bar>` to implement `PartialStorageInfoTrait`
```

Example include https://github.com/paritytech/substrate/pull/12485 and https://github.com/paritytech/substrate/pull/12537 (search for `pallet_ui` in the files).  
No idea what is causing this, but I had it at least three times now.

**Comment by bkchr** [comment]
Hmm and how did you solve these errors? Did you rerun the tests with `TRYBUILD=overwrite`?

**Comment by ggwpez** [comment]
> Did you rerun the tests with `TRYBUILD=overwrite`?

Yes it was "solved" with your `update-rust-stable.sh` script.

**Comment by bkchr** [comment]
Looks like that the number between both tests increased by one. So, maybe some new type that is implemented in these prs that make this number go up? Which is clearly annoying.

**Comment by hirschenberger** [comment]
I did some research and this seems to be resolved in trybuild since 1.0.70 with https://github.com/dtolnay/trybuild/pull/198.
I think an update may solve the issue.
